docs/rsdl/rapid-rsdl-intro.md

```diff
@@ -249,3 +249,44 @@ Functions are invoked using a GET request. Function parameters are passed in the
 | Request                                                                                                                                 | Comment                            |
 | :-------------------------------------------------------------------------------------------------------------------------------------- | :--------------------------------- |
 | GET [http://rapid-pro.org/company/topEmployees?num=10](<https://jetsons.azurewebsites.net/company/Jetsons.Models.topEmployees(num=10)>) | get the company's top 10 employees |
+
+#### Explicit Paths
+
+The set of paths supported by a RAPID service generally do not need to be explicitly enumerated in the RSDL because they are defined by the model. However, in some cases you may want to be explicit about the set of paths supported, either because the service supports only a subset of paths or because you want to specify [capabilities](rapid-rsdl-capabilities.md#annotating-paths) that are specific to that path.
+
+The set of paths supported by a RAPID service can be explicitly enumerated in the RSDL specification:
+
+```rsdl
+paths
```

Comment:
The ABNF requires a colon between `paths` and `{`: is that intended?

Reply:
The colon does stick out because we don't require it for `service` or `type`: propose to remove the colon after `paths`.